project/accounts/admin.py

```diff
@@ -1,8 +1,29 @@
+from categories.models import Category
 from django.contrib import admin
+from notification.models import Notification
+from threads.models import (
+    Activity,
+    Civi,
+    CiviImage,
+    Fact,
+    Rationale,
+    Rebuttal,
+    Response,
+    Thread,
+)
```

Comment:
Register `threads` models from within the `threads` app in the file `threads/admin.py`.

Reply:
Well noted @brylie. Should I go ahead to also register the Category and Notification model from their respective admin?

Reply:
Each model is registered in its respective admin.py, found within the same application directory.

Reply:
Oh @brylie I don't think that was the case, only the account module had an admin file that's why I initially registered all models there. However, I have created an admin file for thread, categories and notification modules, can I go ahead to push for a Pull Request?

Reply:
You can commit those changes in your `register-models branch related to this pull request. No new pull request is necessary.

Reply:
Hello @brylie please I have successfully committed my changes.